Jenkinsfile.sh

```diff
@@ -17,9 +17,16 @@ export ENV_ARG=$environment
 export RC_DIR_ARG="/opt/rocket-chat"
 export S3_BUCKET_ARG="seekingalpha-rocketchat-builds"
 
+if [[ "$version" == rocket.chat-*.tgz ]]
+then
+  versionized_file="$version"
```

Comment:
Rename this variable to `RC_TARBALL_ARG`

* Line 14 defines a convention: Template variables should end in `_ARG`.
* Implied by that is that the variables are in ALL_CAPS
* "versionized file" tells me how it was _generated_, but not what _it is_.  It's a **tarball**.